subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/StronglyTypedConfigurationAttributesResolveIntegrationTest.groovy

```diff
@@ -400,159 +392,4 @@ class StronglyTypedConfigurationAttributesResolveIntegrationTest extends Abstrac
         notExecuted ':b:fooJar', ':b:barJar', ':b:bar2Jar'
     }
 
-    def "can select configuration thanks to producer schema disambiguation"() {
-        given:
-        file('settings.gradle') << "include 'a', 'b'"
-        buildFile << """
-            $typeDefs
-
-            project(':b') {
-               configurationAttributesSchema {
-                  attribute(buildType)
-                  attribute(flavor) {
-                       disambiguationRules.add { details ->
-                            details.closestMatch(details.candidateValues.sort { it.get() }.first())
```

Comment:
Why can't we do this anymore? This disambiguation rule was not using the `AttributeValue` API, so it feels like this should still work.